docs/io.cozy.files.md

```diff
@@ -167,3 +167,48 @@ The `io.cozy.files` doctype has [the standard `cozyMetadata`](https://docs.cozy.
   }
 }
 ```
+
+## `io.cozy.files.versions`
+
+The `io.cozy.files.versions` is used to track old versions of a file content,
+when files versioning is enabled. The attributes of a version are:
+
+- `file_id`: {string} the identifier of the versioned file
```

Comment:
Have you considered putting this in aa `relationships` field?

Reply:
Oh, right, I forgot about that